lms/djangoapps/learner_dashboard/views.py

```diff
@@ -33,11 +33,16 @@ def view_programs(request):
             slug=program['marketing_slug']
         )
 
-    return render_to_response('learner_dashboard/programs.html', {
+    context = {
         'programs': programs,
         'progress': meter.progress,
         'xseries_url': marketing_root if ProgramsApiConfig.current().show_xseries_ad else None,
         'nav_hidden': True,
         'show_program_listing': show_program_listing,
-        'credentials': _get_xseries_credentials(request.user)
-    })
+        'credentials': _get_xseries_credentials(request.user),
+        'disable_courseware_js': True,
```

Comment:
Good question re. documentation @schenedx, I'm not really sure what documentation we currently have around using the pattern library for new pages. @andy-armstrong do we have any?

re. the flags: there is a future story around cleaning up the JS approach a bit, but for now there are the following steps to create a Pattern Library page:
  -  uses_pattern_library flag (for DOM rendering differences - mainly pertaining to the header)
  - disable_courseware_js (to not load the default giant block of JS)
  - set the CSS file to load.